### PR TITLE
Update file_utils.py

### PR DESCRIPTION
**Pull request by JamesDConley**
Replaces poly code with an drastically faster square crop.

On my machine this reduced inference time for one image from 360s to 15s. (~24x speedup).

For images with many detections (many words) it will have even more drastic speedup.

### CODE REVIEW

**Comment by JamesDConley** [comment]
I'm still seeing between 0-30% GPU usage when feeding a sequence of images- so this can definitely be made faster still!

**Comment by gnana70** [comment]
Hi @JamesDConley , Thanks for the PR. Let me go through this and get back to you.

**Comment by JamesDConley** [comment]
Pushed updates. Let me know what you think :smiley:

File: ocr_tamil/craft_text_detector/file_utils.py [hunk 1]
```diff
@@ -141,23 +158,27 @@ def crop_poly(image, poly):
     return cropped
 
 
-def export_detected_region(image, poly, rectify=True):
+def export_detected_region(image, poly, method="rectify"):
```
Comment (gnana70) [inline ocr_tamil/craft_text_detector/file_utils.py:161]: Please change this default to crop  and you are good to g

Comment (gnana70) [reply ocr_tamil/craft_text_detector/file_utils.py:161]: @JamesDConley

Comment (JamesDConley) [reply ocr_tamil/craft_text_detector/file_utils.py]: Updated!

**Pull request merged**